Index.html

```diff
@@ -38,7 +38,137 @@ <h2 class="greenheader">
             <li><a href="https://github.com/CindyDorantes"><img src="./Images/github.png" alt="GitHub icon" class="socialicon"></a></li>
             <li><a href="https://angel.co/"><img src="./Images/angellist.png" alt="AngelList icon" class="socialicon"></a></li>
         </ul>
+    </section>
+        <p><img src="./Images/arrow_button.png" alt="arrow button" class="arrow_button"></p>
+    <!--here begins the 'Works' section-->
+    <section class="works">
+        <h2 class="works-heading">My recent Works <br><img src="./Images/Indicator.png" alt="green line divisor"></h2>
+        <span><img src="./Images/Indicator.png" alt="green line divisor"></span>
         
+        <!--First work-item-->
+        <div class="work-item01">
+            <div class="work-image"></div>
+            <div class="work-content">
+                <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+                <ul class="cards">
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>
+                <button type="button" class="see-project">See project</button>                  
+            </div>
+        </div>
```

Comment:
- [ ] [OPTIONAL] Please make changes! 
- Fix your `See Project` button, because your button is not as similar as the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). 

- Your project opens only the same size in the mobile version, resulting in a white field on the left when expanded from 300 px to 800 px. My advice to you would be to ask you to fix this problem. Let's learn [additional information](https://youtu.be/yU7jJ3NbPdA).

![image](https://user-images.githubusercontent.com/63915024/172029541-5e8598cd-5cd9-4dac-b10b-7913002ba94e.png)